src/components/TabNavLink/TabNavLink.tsx

```diff
@@ -0,0 +1,19 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  to: string,
+  text: string,
+};
+
+export const TabNavlink: FC<Props> = ({ to, text }) => (
+  <NavLink
+    to={to}
+    className={({ isActive }) => (
+      classNames({ 'is-active': isActive })
+    )}
```

Comment:
If I'm not mistaken we are changing class only in `li` element.
![image](https://user-images.githubusercontent.com/63786090/220886837-24e93b96-ffdd-4c61-bdfa-8b382047ba7e.png)

Reply:
yeah, @Avichez is right, you have class `is-active` in li element, here you can use <Link>